main.py

```diff
@@ -69,6 +69,7 @@ def setup_mqtt_client(configuration: Configuration) -> MqttClient:
 
 
 def main():
+    print("new version is running!")
```

Comment:
I like having more verbose mode, but I am not that comfortable with plain prints. I would rather do that using a logging library instead

Reply:
I recommend to avoid including this text here and leave the logging as a separate task to focus on the new library